browser.go

```diff
@@ -22,6 +22,9 @@ import (
 // the browser process runner, WebSocket clients, associated targets, and
 // network, page, and DOM events.
 type Browser struct {
+	// next is the next message id.
+	next int64
```

Comment:
please add a comment here, like:

```
// NOTE: needs to be 64-bit aligned for 32-bit targets too, so be careful when moving this field.
// This will be eventually done by the compiler once https://github.com/golang/go/issues/599 is fixed.
```

Reply:
Done!